pkg/ansible/runner/runner.go

```diff
@@ -307,7 +309,15 @@ func (r *runner) makeParameters(u *unstructured.Unstructured) map[string]interfa
 		spec = map[string]interface{}{}
 	}
 
-	parameters := paramconv.MapToSnake(spec)
+	parameters := map[string]interface{}{}
+	for k, v := range spec {
+		parameters[k] = v
+	}
+
+	if r.snakeCaseParameters {
+		parameters = paramconv.MapToSnake(spec)
+	}
```

Comment:
Is not better use the else here? is not the above for unnecessary if is to snake?

Reply:
Yeah definitely, good catch